string head
int32 state_1
int32 state_2
int32 f_satellite
int32 b_satellite
